UKHomeOffice/sgmr-service
# SPCR-510 Update htmlFor to match input id

### AC / Description of changes

/save-voyage/select-employees `htmlFor` label for the checkbox did not match the input `id`, now it does


### To Test

- Run axe devtools on this page, should not see an error about missing label
- click on the label for a checkbox, it should select the checkbox


### Notes

---

**Remember to merge to feature branches not `master`.**
